news/_posts/2021-05-31-prestashop-1-7-8-0-beta-release.md

```diff
@@ -0,0 +1,233 @@
+---
+layout: post
+title:  "PrestaShop 1.7.8.0 beta is open for feedback!"
+subtitle: "You can download it and test it"
+date:   2020-05-31 17:00:00
+authors: [ PrestaShop ]
+icon: icon-leaf
+tags:
+ - version
+ - beta
+ - minor
+ - releases
+ - 1.7.8
+---
+
+The first Beta version for PrestaShop 1.7.8.0 is now ready for you to test!
+
+We are very happy to announce today the release of the beta version of 1.7.8.
+This version includes many features and bug fixes, we count on you to test every nook and cranny and report any bugs you encounter.
+
+In the new features: we have the redesign of the translation system that facilitates export and translation of your modules and themes, a new custom header for multistore fans and many other improvements.
+
+//Focus on the contribution (stats)
+
+We need the help of all the community to test this version. It integrates more than 1,372 Pull Request merged and 241 bugs fixed including 17 major bugs and 4 Topwatchers bugs (an issue that are followed by more than 5 people). 
+
+**Remember, this beta version is pre-release software. Do not use it in your production shop!**
+
+## What’s new?
+
+### New features & improvements
+
+#### Translations
+
+- The translation has been revamped, adding the possibility to export Back-Office, Front-Office, eMail and Modules translations.
+
+![the new translation export](/assets/images/2021/translation-export.png)
```

Comment:
The screenshot shows a French Back-Office 🥖 🍷

Reply:
good point ! I need to change most of the screenshots